class-components/src/views/SearchView/SearchView.tsx

```diff
@@ -0,0 +1,65 @@
+import { FunctionComponent, useEffect, useState } from 'react';
+import Search from '../../components/Search/Search';
+import { Person } from '../../helpers/interfaces';
+import List from '../../components/List/List';
+import Loader from '../../components/Loader/Loader';
+import ErrorButton from '../../components/ErrorButton/ErrorButton';
+import ErrorBoundary from '../../components/ErrorBoundary/ErrorBoundary';
+import Pagination from '../../components/Pagination/Pagination';
+import { Outlet, useNavigate, useParams } from 'react-router-dom';
+
+import './searchView.css';
+
+const SearchView: FunctionComponent = () => {
+  const { page, elementId } = useParams();
+  const navigate = useNavigate();
+  const [elements, setElements] = useState<Person[]>([]);
+  const [elementsCount, setElementsCount] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+  const currentPage = page ? +page : 1;
+
+  useEffect(() => {
+    if (!page) navigate('/search/1');

```

Comment:
```suggestion
    if (!page) navigate(HOME_PAGE); 
```
или типа того. можно или в енам вынести или в константы. наверное в твоем случае лучше в константы, типа

const BASE_PATH = '/search'
const HOME_PAGE = '/search/1'

ну или в енам